package.json

```diff
@@ -34,6 +34,8 @@
     "react": "^0.13.0"
   },
   "dependencies": {
+    "create-react-class": "^15.5.1",
+    "prop-types": "^15.5.6",
```

Comment:
No, it should be in `dependencies`.

I'm happy to add maintainers who won't change the API (I don't think it's worth changing it at this point to avoid churn in a very simple library). If you'd like to help with maintenance let me know